--- paddleocr.py.orig	2024-07-19 11:18:02.441013507 +0000
+++ paddleocr.py	2024-07-19 11:18:11.996947548 +0000
@@ -727,7 +727,7 @@
             for idx, img in enumerate(imgs):
                 img = preprocess_image(img)
                 dt_boxes, rec_res, _ = self.__call__(img, cls, slice)
-                if not dt_boxes and not rec_res:
+                if not dt_boxes.any() and not rec_res:
                     ocr_res.append(None)
                     continue
                 tmp_res = [[box.tolist(), res] for box, res in zip(dt_boxes, rec_res)]
