--- paddleocr.py.orig	2024-02-02 10:48:50.843676044 +0100
+++ paddleocr.py	2024-02-02 10:52:45.109931696 +0100
@@ -671,7 +671,7 @@
             for idx, img in enumerate(imgs):
                 img = preprocess_image(img)
                 dt_boxes, elapse = self.text_detector(img)
-                if not dt_boxes:
+                if not dt_boxes.any():
                     ocr_res.append(None)
                     continue
                 tmp_res = [box.tolist() for box in dt_boxes]
